cypress_test/integration_tests/desktop/impress/annotation_spec.js

```diff
@@ -15,7 +15,7 @@ describe('Annotation Tests', function() {
 		if (Cypress.env('INTEGRATION') === 'nextcloud') {
 			desktopHelper.hideSidebarIfVisible();
 		} else if (Cypress.env('USER_INTERFACE') === 'notebookbar') {
-			cy.get('#ModifyPage').click();
+			cy.get('#ModifyPage1').click();
```

Comment:
please use `.unoModifyPage` instead, we changed uno buttons identifiers to be unique, but it is not guaranted that it will always have '1' here :)

so instead of #XXX always use .unoXXX for unobuttons